script.js

```diff
@@ -0,0 +1,44 @@
+let fisrtName = document.getElementById('fisrtName');
+let lastName = document.getElementById('lastName');
+let email = document.getElementById('email');
+let age = document.getElementById('age');
+let phone = document.getElementById('phone');
+let myBtn = document.getElementById('btn');
+let errorMessage = document.getElementsByTagName('p')[0];
+console.log(errorMessage);
+const validateData = (input) => {
+    if(input==="age"){
+        if(age.value < 18) {
+            errorMessage.innerText ="age must be greater than 18"
+        }
+        else if (age.value > 100) {
+            errorMessage.innerText = "age must be less than 100"
+        }
+        else errorMessage.innerText = " "
+    }
+    else if (input === "phone") {
+        if (phone.value.length < 10) {
```

Comment:
Can we replace these condition to below code then you do not need to check 2 conditions `phone.value.length < 10` and `phone.value.length  > 10 `

```
if(phone.value.length !== 10 ){
     errorMessage.innerText = "Please enter the valid number"
     }
     else {
         else errorMessage.innerText = ""
}